index.js

```diff
@@ -1,9 +1,9 @@
-import bookData from './modules/Books';
+import Books from './modules/Books.js';
```

Comment:
- [ ] The project can't be run, please consider implementing the run script into ```packeage.json```